akka-http2-support/src/test/scala/akka/http/impl/engine/http2/parsing/HttpRequestHeaderHpackDecompressionSpec.scala

```diff
@@ -42,6 +42,7 @@ class HttpRequestHeaderHpackDecompressionSpec extends AkkaSpec with ScalaFutures
       request.method should ===(HttpMethods.POST)
     }
     "decompress given CONTINUATION Headers frames" in {
+      pending // FIXME: this test seems to be bogus?!?
```

Comment:
Is it? I'll check.
I thought that's how headers spread into more frames look like